be/src/vec/exec/format/orc/vorc_reader.cpp

```diff
@@ -607,21 +607,28 @@ TypeDescriptor OrcReader::_convert_to_doris_type(const orc::Type* orc_type) {
     case orc::TypeKind::TIMESTAMP_INSTANT:
         return TypeDescriptor(PrimitiveType::TYPE_DATETIMEV2);
     case orc::TypeKind::LIST: {
+        // Use Object PrimitiveType to construct TypeDescriptor also need init containers_nulls with the default value of bool
         TypeDescriptor list_type(PrimitiveType::TYPE_ARRAY);
         list_type.children.emplace_back(_convert_to_doris_type(orc_type->getSubtype(0)));
```

Comment:
better add a function like:
`TypeDescriptor::add_sub_type(type, bool is_nullable = true);`

contains_nulls should be true as default.

Reply:
ok